dev/how-to/maintain/tidb-in-kubernetes/monitoring.md

```diff
@@ -0,0 +1,97 @@
+---
+title: 监控系统部署
+category: how-to
+---
+
+# 监控系统部署
+
+基于 Kubernetes 环境部署的 TiDB 集群监控可以大体分为两个部分：对 TiDB 集群本身的监控、对 Kubernetes 集群及 TiDB Operator 的监控。本文将对两者进行简要说明。
+
+## TiDB 集群的监控
+
+TiDB 通过 Prometheus 和 Grafana 监控 TiDB 集群。在通过 TiDB Operator 创建新的 TiDB 集群时，对于每个 TiDB 集群，会同时创建、配置一套独立的监控系统，与 TiDB 集群运行在同一 Namespace, 包括 Prometheus 和 Grafana 两个组件。
+
+监控数据默认没有持久化，如果由于某些原因监控容器重启，已有的监控数据会丢失。可以在 `values.yaml` 中设置 `monitor.persistent` 为 `true` 来持久化监控数据。开启此选项时应将 `storageClass` 设置为一个当前集群中已有的存储，并且此存储应当支持将数据持久化，否则仍然会存在数据丢失的风险。
+
+在 [TiDB 集群监控](monitor-a-cluster.md)中有一些监控系统配置的细节可供参考。
+
+### 查看监控面板
+
+可以通过 `kubectl port-forward` 查看监控面板：
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl port-forward -n ${namespace} svc/${releaseName}-grafana 3000:3000 &>/tmp/portforward-grafana.log
+```
+
+然后在浏览器中打开 http://localhost:3000，默认用户名和密码都为 `admin`。
```

Comment:
This is to avoid the possible display error at the PingCAP website.

```suggestion
然后在浏览器中打开 [http://localhost:3000](http://localhost:3000)，默认用户名和密码都为 `admin`。
```